.github/workflows/push-to-prod-draft.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           FIRMWARE_SHORT_HASH=$( echo ${GITHUB_SHA:0:7} )
           echo "FIRMWARE_SHORT_HASH=$FIRMWARE_SHORT_HASH" >> $GITHUB_ENV
       - name: Random sleep (balena API workaround)
-        run:  sleep $[ ( $RANDOM % 180 )  + 1 ]s
+        run:  sleep $[ ( $RANDOM % 180 ) + ( $RANDOM % 180 ) + 1 ]s
```

Comment:
the idea was to make it a wider time range, because we keep getting deploy failures again (possibly due to adding more variants to the deploy and so the API keys are being used simultaneously again) - which would hopefully make it more reliable.

i did this when i was tired though, and my sleepy thinking was that adding two randomly generated numbers together would be different to a random assignment of double the number.